libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -562,6 +567,14 @@ void AP_Mount::set_angle_targets(uint8_t instance, float roll, float tilt, float
     _backends[instance]->set_angle_targets(roll, tilt, pan);
 }
 
+
+#define VEHICLE_YAW_SCALE 1.0f
```

Comment:
Why is this macro necessary? Should it be a parameter?

Reply:
Changed it to a parameter